### ISSUE TITLE
Move cursor one back please

### ISSUE THREAD

**Issue by mentalisttraceur** [casouri/vundo#99]
Currently, the point is placed after the node glyph instead of before.

When using Emacs in some terminals, the cursor can't be hidden, and can't be restyled, especially not per-buffer (and doing it globally for the whole terminal usually makes for worse UX over-all).

One real-world example of where you basically have to use such a terminal is in on Android, in Termux. Technically that terminal supports some standard control sequences to restyle the cursor (neovim manages to do it), but Emacs doesn't seem to know how to do it? Anyway, for me this is a huge part of my daily Emacs usage.

So it's a super minor aesthetic issue, but it seems like it would be a strict improvement, and wouldn't have any effect in displays where the cursor can be made invisible.

**Comment by mentalisttraceur** [comment]
![Screenshot_20240214-235846](https://github.com/casouri/vundo/assets/6531708/5bb72548-a2f6-40f4-a221-56a57ce9e11d)

Here's an example of what it ends up looking like currently (compact, Unicode glyphs, in Termux on Android).

I think it would be strictly better if the point was left one glyph to the left, because when the cursor can't be hidden it would at least degrade to this more intuitive view:

![Screenshot_20240215-000221](https://github.com/casouri/vundo/assets/6531708/1e20b384-e840-485a-9628-fc9d9dd41053)

**Comment by mentalisttraceur** [comment]
I could actually see how this can be a matter of taste, and looking at it, maybe I actually prefer the way it is?

So I'm changing my recommendation to making it a config variable. Or maybe we can just figure out a function that users could add to one of the existing hooks to achieve this?

**Comment by mentalisttraceur** [comment]
(This might be related to #49 , but I don't have time to double-check right now, sorry.)

**Comment by casouri** [comment]
Makes sense, if someone care about this deeply enough and make a pr, I'm happy to merge it.
